feature-libs/product-configurator/rulebased/core/state/effects/configurator-basic.effect.ts

```diff
@@ -269,7 +269,8 @@ export class ConfiguratorBasicEffects {
                 map((currentGroupId) => {
                   const groupIdFromPayload =
                     this.configuratorBasicEffectService.getFirstGroupWithAttributes(
-                      payload
+                      payload,
+                      currentGroupId?.startsWith('CONFLICT')
```

Comment:
Please add a test for updateConfigurationSuccess and current group as conflict group, I think this is not covered

Reply:
done